sycl/test/basic_tests/image_accessor_readsampler.cpp

```diff
@@ -1,4 +1,5 @@
 // UNSUPPORTED: cuda
+// UNSUPPORTED: level0
```

Comment:
You have only test changes in this PR and no functional change in the test. Why this test was marked us unsupported for level0?